packages/distutils/meta.yaml

```diff
@@ -5,9 +5,10 @@ package:
     - always
   top-level:
     - distutils
+source:
+  sha256: 065040198fde6fd7229b8c0538fc8b848824e35930e3904f46c9e90e9f0f1a3f
+  url: https://github.com/python/cpython/archive/refs/tags/v$(PYVERSION).tar.gz
```

Comment:
Unfortunately, sha256 hash needs to be updated manually when we update CPython version...

Reply:
We could put it in an env variable otherwise? For 7 manual replacements, it's probably worth it...

Reply:
Sound good.